lib/membrane/rtp/jitter_buffer/buffer_store.ex

```diff
@@ -211,19 +210,16 @@ defmodule Membrane.RTP.JitterBuffer.BufferStore do
     end
   end
 
-  defp add_record(%__MODULE__{heap: heap, set: set} = store, %Record{} = record) do
+  defp add_record(%__MODULE__{heap: heap, set: set} = store, %Record{} = record, record_epoch) do
     if set |> MapSet.member?(record.index) do
       store
     else
       %__MODULE__{store | heap: Heap.push(heap, record), set: MapSet.put(set, record.index)}
       |> update_end_index(record.index)
+      |> update_roc(record_epoch)
     end
   end
 
-  defp bump_prev_index(%{prev_index: prev, rollover_count: roc} = store)
-       when rem(prev + 1, @seq_number_limit) == 0,
-       do: %__MODULE__{store | prev_index: prev + 1, rollover_count: roc + 1}
-
   defp bump_prev_index(store), do: %__MODULE__{store | prev_index: store.prev_index + 1}
```

Comment:
- I think it doesn't deserve a separate function now, or at least give it `update` prefix as the others have
- Let's rename this `prev_index` to something more meaningful, or at least less misleading :P Maybe `start_index`?